docs/src/pages/examples/rag/filter-store.mdx

```diff
@@ -0,0 +1,164 @@
+---
+title: "Example: Vector Store with Filters | RAG | Mastra Docs"
```

Comment:
Let's call this Hybrid Vector Search

<img width="648" alt="image" src="https://github.com/user-attachments/assets/0049a846-9a36-467e-b23e-0f231db950db" />

Reply:
Include some of that context in the header of the doc too